src/test/resources/logback.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
:+1: for restoring this in the correct place as logback initialization is required for some of the `DebugToolsTest` unit tests